_posts/2018-01-13-the-new-css-layout.md

```diff
@@ -0,0 +1,26 @@
+---
+title: The New CSS Layout
+custom_type: post
+review:
+  type: book
+  book:
+    author: Rachel Andrew
+  rating: 4
+  vendor: A Book Apart
+  url: https://abookapart.com/products/the-new-css-layout
+categories: [design, web design]
+---
+
+If you've been working on the web for the past few years, you've noticed how quickly our layout tools have improved. I gave four talks last year on CSS Grid. In just 2017, the feature went from being behind flags to having support in most modern browsers.
+
+*The New CSS Layout* by Rachel Andrew provides a comprehensive summary of the history of layout, what tools we have at our disposal right now, and where we're heading. Her explanation of why and how to get involved in the making of CSS is the rally cry I think our industry needs.
+
+While some complain incessantly about their frustrations with CSS and how it's somehow a "better idea" to write CSS in JS, Rachel provides us with a constructive way to better CSS. Turns out, that to be part of creating the future of CSS, you can create an issue on GitHub and describe your particular use case of a new feature. In other words, you have to truly care. Writing a whiny article on Medium with a click-bait headline doesn't count.
+
+Rachel dedicates a good portion of this book teaching us how to use the tools we currently have, the right way. As part of my own talk on CSS Grid addresses, we've been using features like floats and even flexbox as hacks for what CSS Grid came to fix.  A crucial part in us improving the layouts we create will be rewriting the parts of our brain that had learned the wrong way.
```

Comment:
"use the tools we currently have, the right way." Strange or unnecessary use of comma.